### PR TITLE
ADAPT-2989: More card tweaks

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Misc card tweaks (called out below)

# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. See event cards https://deploy-preview-122--stanford-alumni.netlify.app/test/yvonne/test-events/
2. For the non minimal cards, hover or tab to a card, the shadow should deepen. The tab label should change to a brighter red and the image zoom will happen when kayboard focused (matches hover state).
3. Scroll down to the bottom dark minimal cards - check that it looks ok.
4. Check that all the event detail icons look ok (the calendar, map pin/computer and organizer icons).
5. See story cards https://deploy-preview-122--stanford-alumni.netlify.app/test/yvonne/test-story-cards/
6. Please perform the same test as above and verify they all look ok.
7. see perk cards https://deploy-preview-122--stanford-alumni.netlify.app/test/yvonne/test-perks/
8. Use keyboard to tab to the cards, see that the image zoom happens on focus.
7. Look at code

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/components/content-types/event/event.js [hunk 1]
```diff
@@ -75,24 +76,44 @@ const Event = ({
...
+
   let headlineSize = "su-type-1";
 
   if (isBigHeadline) {
     headlineSize = dcnb("lg:su-type-2 xl:su-type-3", headlineSize);
   }
 
   const iconClasses =
-    "su-inline-block su-flex-shrink-0 su-mt-01em su-mr-06em su-w-[1.1em]";
+    "su-inline-block su-flex-shrink-0 su-mt-2 md:su-mt-3 su-mr-06em su-w-[1em]";
```
Comment (yvonnetangsu) [inline src/components/content-types/event/event.js:116]: Per Kerri's request, reduced the event card detail icons from 1.1em to 1em, and adjust positioning accordingly.

File: src/components/content-types/event/event.js [hunk 2]
```diff
@@ -75,24 +76,44 @@ const Event = ({
   }
 
   let wrapperClasses =
-    "su-rs-pb-3 su-bg-white su-border su-border-solid su-border-black-30-opacity-40 su-bg-clip-padding su-shadow-sm focus-within:su-shadow-md hover:su-shadow-md";
+    "su-rs-pb-3 su-bg-white su-border su-border-solid su-bg-clip-padding su-shadow-sm focus-within:su-shadow-md hover:su-shadow-md su-backface-hidden";
```
Comment (yvonnetangsu) [inline src/components/content-types/event/event.js:79]: Use focus-within for keyboard focus shadow change for event cards

File: src/components/content-types/event/event.js [hunk 3]
```diff
@@ -33,6 +33,7 @@ const Event = ({
   headingLevel,
   tabText,
   hideTab,
+  isDark,
```
Comment (yvonnetangsu) [inline src/components/content-types/event/event.js:36]: Add dark theme support for event cards

File: src/components/content-types/event/event.js [hunk 4]
```diff
@@ -124,7 +147,7 @@ const Event = ({
                   filename={filename}
                   imageFocus={imageFocus}
                   size="vertical"
-                  className="su-w-full su-h-full su-object-cover su-transition-transform su-transform-gpu group-hover:su-scale-[1.03]"
+                  className="su-w-full su-h-full su-object-cover su-transition-transform su-transform-gpu group-hover:su-scale-[1.03] group-focus-within:su-scale-[1.03]"
```
Comment (yvonnetangsu) [inline src/components/content-types/event/event.js:150]: Use group-focus-within to create image zoom effect for card on focus

File: src/components/simple/tabLabel.js [hunk 5]
```diff
@@ -5,7 +5,7 @@ import { dcnb } from "cnbuilder";
 const TabLabel = ({ text, srText, classes, ...props }) => {
   // Focus isn't on the card itself since we are using the stretched-link class for accessibility, so no need for group-focus styles
   const interactionClasses =
-    "group-hover:su-bg-none group-hover:su-bg-digital-red-light";
+    "group-hover:su-bg-none group-hover:su-bg-digital-red-light group-focus-within:su-bg-none group-focus-within:su-bg-digital-red-light";
```
Comment (yvonnetangsu) [inline src/components/simple/tabLabel.js:8]: use group-focus-within on the tab label, so it changes color when the card stretched link is keyboard focused

**Review by rebeccahongsf — APPROVED** [review]
Preview and code looks good to me 😄  Merge away @yvonnetangsu !

**Comment by yvonnetangsu** [comment]
@rebeccahongsf Thank you for reviewing! I know I owe you Decanter reviews 🙏🏼

**Pull request merged**